week-1/G-strings-methods/exercise2.js

```diff
@@ -1,3 +1,9 @@
-var name = " Daniel  ";
-
-console.log(message);
+var name = "Blaise    ";
+var mynamelenght = name.length;
+console.log(
+  "My name is " +
+    name.trimRight() +
+    " and my name is " +
+    mynamelenght +
```

Comment:
`mynamelenght` hasn't been trimmed so this value is incorrect. Next time, try trimming the name earlier in your code so you know it's done, then you'll have the correct string for the rest.